UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -164,13 +164,23 @@ void ApplyConfig()
 			var telepathy = GetComponent<TelepathyTransport>();
 			if (telepathy != null)
 			{
-				telepathy.port = (ushort) config.ServerPort;
+				telepathy.port = (ushort)config.ServerPort;
 			}
 
 			var ignorance = GetComponent<Ignorance>();
 			if (ignorance != null)
 			{
-				ignorance.port = (ushort) config.ServerPort;
+				ignorance.port = (ushort)config.ServerPort;
+
+			}
+		}
+		if (string.IsNullOrEmpty(config.BindAddress) == false)
+		{
+			var ignorance = GetComponent<Ignorance>();
```

Comment:
Must be a generic way of setting this?

Reply:
I can move the variable declaration outside if that's what you mean?

Reply:
@Bod9001 do you mean Telepathy and Ignorance extend the same class?

Reply:
Doesn't ignorance use a generic networking layer so mirror works with it, that's what I mean

Reply:
No, it can't be abstracted to that. Ignorance is the transport, which is below mirror. Ignorance does the listening (with ENet), Mirror is just the protocol.

Reply:
ok cool